src/Symfony/Bundle/WebProfilerBundle/Resources/views/Profiler/base_js.html.twig

```diff
@@ -349,7 +349,7 @@
                         }
 
                         tabNavigation[j].addEventListener('click', function(e) {
-                            var activeTab = e.target || e.srcElement;
+                            var activeTab = this;
```

Comment:
Good point @stof. When using native `addEventListener` I'm quite sure that `this` references the target element in all browsers, but I just notice `Sfjs.addEventListener` falls back to `attachEvent` if `addEventListener` isn't avialable (especially for IE<9) since #13636. In listeners attached with `attachEvent` `this` doesn't references the target element, without further effort.

When looking too lines below, it turns out that the navigation tab listener already considers the case of embedded html by traversing the DOM tree up to the expected tag. I overlooked that before. I will update this PR to revert this listener and use the same technique for the search button listener.